### ISSUE TITLE
Replace embedded text in home page image with text

### ISSUE THREAD

**Issue by EllisBarnwellCC** [carboncoop/powershaper-tracker#128]
Embedding text in images is bad practice from accessibility point of view. Also it also non-responsive so only works at desktop level, and description of the site is not readable on mobile (or tablet).

![Screenshot_20230327-133424](/uploads/b8a1e4d0fb3cf672b3dcfe5d6d27a252/Screenshot_20230327-133424.png)

*Imported from GitLab on 2025-09-30. Originally created by Matt Fawcett on 2023-03-27T14:28Z.*
